plugins/console/plugin.c

```diff
@@ -76,6 +76,7 @@ plugin_iface_init (
 
   iface->name = "XMPP console";
   iface->version = PACKAGE_VERSION;
+  iface->sidecar_interfaces = sidecar_interfaces;
```

Comment:
I'm not sure about this, why did you add it?

Reply:
this eventually a fix for this sidecar plugin as the reason the static variable was unused is because this line was missing. The sidecar though is rather example so no one ever cared I presume whether it actually worked. I can probably move it to separate commit as there's also related python script (client side of the sidecar interface - to test its functionality) which also needs similar one line fix to make it work and the commit title doesn't match the purpose